README.md

```diff
@@ -13,27 +13,27 @@
 Proxy utility to bundle a batch of calls in one request.
 Using the batchelor utility reduces HTTP overhead, network round-trip delay time and helps to keep your API design clean.
 
-###Features:
+
+## Features
+
 * Server side parallel request processing.
 * Persistent request for Web Socket facade.
 
-### NMP Package
-npm install batchelorjs --save
 
-### API
-* [`configure(options)`](#configure)
-* [`execute(batch, callback)`](#execute)
-* [`stop(options)`](#stop)
-* [`Events`](#events)
+## Installation
+
+    npm install batchelorjs --save
+
 
-### Methodology/Examples
+## API
 
-* [`batch`](#batch)
-* [`request`](#request)
-* [`examples`](#examples)
+* [`configure(options)`](#configureoptions)
```

Comment:
I would remove the list (keep the `API` header) and instead add a table of content at the beginning.
You can use [doctoc](https://www.npmjs.com/package/doctoc) to auto generate it